src/commands/debug.json

```diff
@@ -15,6 +15,44 @@
             "LOADING",
             "STALE",
             "PROTECTED"
-        ]
+        ],
+        "reply_schema": {
```

Comment:
please add a description to all of the subschemas

Reply:
on second thought, i just remembered that DEBUG is the only command we decided not to get a schema, so you can revert this file and make an exception for DEBUG in the validator script